### PR TITLE
PR for question-3

### PR DESCRIPTION
**Pull request by Yogesh-Git-Tagline**

### CODE REVIEW

File: eg3.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+"""You are given a list of sentences. Create a word tree from each sentence and find how many times the word is appeared in other word trees."""
+
+#list created with sentences
+sentences = ["My name is Ram", "He is a good person", "You should be careful while coding",
+             "He can do better", "The person is mysterious", "Jay Shree Ram", "It is my pen"]
+
+
+SplitSentences = []
```
Comment (himanshu-suthar-tagline) [inline eg3.py]: For each PR, please make sure you declare the variable names in smaller cases, as it is the correct way to declare the names. See the python standard rules for naming conventions.

File: eg3.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+for i in sentences:
```
Comment (himanshu-suthar-tagline) [inline eg3.py]: Instead of `i`, give some proper name for the loop variable.

File: eg3.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
+"""You are given a list of sentences. Create a word tree from each sentence and find how many times the word is appeared in other word trees."""
+
+#list created with sentences
+sentences = ["My name is Ram", "He is a good person", "You should be careful while coding",
+             "He can do better", "The person is mysterious", "Jay Shree Ram", "It is my pen"]
+
+
+split_sentences = []
+for sentence in sentences:
+    split_sentences.append(sentence.split())  #spliting each sentence of list
```
Comment (himanshu-suthar-tagline) [inline eg3.py:10]: You can use .extend to avoid for loop below. Please check that.

File: eg3.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+for sentence in split_sentences:
+    for element in sentence:
+        merge_items.append(element)    #store the each item from splited list
+
+print(split_sentences)      #printing the splited sentences with list
+
+items = []           #item list for store items without duplicate items
+for item in merge_items:
+    if item not in items:
+        items.append(item) #remove the duplicate items using this loop and store the in another list
```
Comment (himanshu-suthar-tagline) [inline eg3.py]: You can use combination of set and list to avoid this loop as well.

File: eg3.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+item_occurence = []      #item_occurence list for store the occurence of each item from TempList
+for item in items:
+    item_occurence.append(merge_items.count(item))  #this process is for store the each item occurence in list
```
Comment (himanshu-suthar-tagline) [inline eg3.py]: Check the python `collections.Counter`

File: eg3.py (hunk 3 above)
Comment (Yogesh-Git-Tagline) [reply eg3.py:10]: yes, i used .extend method insted of below for loop(see line no.14)

File: eg3.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+for sentence in sentences:
+    split_sentences.append(sentence.split())  #spliting each sentence of list
+
+merge_items = []
+for sentence in split_sentences:
+        merge_items.extend(sentence)    #store the each item from splited list
+
+print(split_sentences)      #printing the splited sentences with list
+
+result=Counter(merge_items)        #collections.Counter to count each word from merge_item list
```
Comment (Yogesh-Git-Tagline) [inline eg3.py:18]: i used Collection.counter module as you told me

**Comment by himanshu-suthar-tagline** [comment]
You can merge the PR after resolving conflicts.

**Pull request merged**